src/themes/material/templates/journal/editorial_team.html

```diff
@@ -37,13 +37,12 @@ <h2>{{ group.name }}</h2>
                                     </small>
                                 </p>
                             {% endif %}
-                            <p>
                             {% include "elements/journal/editorial_social_content.html" with user=member.user %}
-                            </p>
-
                         </div>
                     </div>
                 </div>
+                {% if forloop.counter|divisibleby:4 %}</div>
```

Comment:
Needs to account for column sizes (we use m3 but also s12)

Reply:
at s12 the row switch makes no difference to the display so we're ok.